validation_config.toml

```diff
@@ -49,7 +49,7 @@ select = [
     "PR101"
 ]
 
-warning = []
+warning = ["DS107"]
```

Comment:
We can revert it once this [issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-11613) is resolved.